### ISSUE TITLE
asciinema doesn't capture ncurses box

### ISSUE THREAD

**Issue by adder46** [asciinema/asciinema-player#107]
I recorded a [snake game session](https://github.com/adder46/snake/blob/master/snake.gif), however it seems that asciinema doesn't capture ncurses box.

It would be nice if asciinema could do that.

**Comment by adder46** [comment]
> Do you have example recording link (on asciinema.org) that demonstrates the problem?

The recording I linked is made using asciinema and converted to .gif via `asciicast2gif`.

**Comment by sickill** [comment]
Yes, I understand, but I have nothing to compare the gif too... No idea how it looks in original :)

**Comment by adder46** [comment]
I have made a [minimal, complete and verifiable example](https://gist.github.com/adder46/e4bac722ea4722ea3f6bd761bc462dbc) which demonstrates the issue. As you can see, the border is display correctly in the [recording](https://asciinema.org/a/vQwSCQOOdbVpdupfrJSFBoEHe) but once the recording is converted to .gif the border is gone. Thus, I think the actual problem is in `asciicast2gif`.
